parser/SqlLexer.lpp

```diff
@@ -120,6 +120,8 @@ unsigned_numeric_literal {exact_numeric_literal}|{approximate_numeric_literal}
 "between"          return TOKEN_BETWEEN;
 "bigint"           return TOKEN_BIGINT;
 "bit"              return TOKEN_BIT;
+"bloomfilter"      return TOKEN_BLOOM_FILTER_INDEX;
```

Comment:
In most cases (except for keywords that are synonyms), the token name should follow from the actual keyword, e.g. `TOKEN_BLOOM_FILTER` without the trailing "_INDEX"